pkg/config/flowaggregator/config.go

```diff
@@ -73,3 +75,18 @@ type APIServerConfig struct {
 	// TLS min version.
 	TLSMinVersion string `yaml:"tlsMinVersion,omitempty"`
 }
+
+type ClickHouseConfig struct {
+	// Username is the username for ClickHouse client.
+	Username string `yaml:"username,omitempty"`
+	// Password is the username for ClickHouse client.
```

Comment:
username -> password

Reply:
this config is obsolete as username and passwords are now read from ENV set from k8s secret.

Reply:
Sure, but we should still correct the word here before we remove it.

Reply:
done